app/editor/src/components/tab-control/TabControl.tsx

```diff
@@ -0,0 +1,35 @@
+import { FaExternalLinkAlt, FaExternalLinkSquareAlt, FaRegClone } from 'react-icons/fa';
+import { useApp } from 'store/hooks';
+import { Show } from 'tno-core';
+
+import { NavigateOptions } from './constants';
+
+export const TabControl: React.FC = () => {
```

Comment:
Simple component to control the tab behaviour.

Reply:
I dont quite feel like that the header of the row is the right place for this component, but I can see how having it there puts it front and center to the user.  Bobbi might have some say on it when she sees it in play.

Reply:
I 100% agree.  Just couldn't think of a better location without the whole toolbar resulting in a wrap...